syft/frameworks/torch/pointers/pointer_tensor.py

```diff
@@ -307,6 +307,24 @@ def fix_prec(self, *args, **kwargs):
 
     fix_precision = fix_prec
 
+    def float_prec(self, *args, **kwargs):
+        """
+        Send a command to remote worker to transform a fix_precision tensor back to float_precision
+
+        Returns:
+            A pointer to a Tensor
+        """
+
+        # Send the command
+        command = ("float_prec", self, args, kwargs)
+
+        response = self.owner.send_command(self.location, command)
+
+        # For some reason the response has no Wrapper. Must be manually added here
```

Comment:
Hmm, I don't like that you have to wrap the response for this specific case but not for the others... I think it's worth exploring why this is the case...

Reply:
I've found why, but I'm not sure how to fix it. the definition of `float_prec()` in `interpreters/native.py` is 
```
def float_prec(self):
    return self.child.float_precision()
```
which differs to `fix_prec()` here
```
if self.is_wrapper:
    self.child = self.child.fix_prec(*args, **kwargs)
    return self
```
What I can see is that it is assumed that a float_precision tensor is wrapped, and must return an unwrapped torch tensor. I see two ways of solving this:

1. check if `self.child` is a pointer, in which case return the wrapped tensor
```
def float_prec(self):
    if self.child is PointerTensor:
        self.child = self.child.float_prec()
        return self
    return self.child.float_precision()
```
2. Check that if a torch tensor has a wrapper, then remove the wrapper, as it is useless. 

The first option is the easiest, but I feel is somewhat dirty to have to manually check for type and specify different behavior. The second one might be out of my knowledge on how to implement it.

What do you think?

Reply:
I agree that 2. looks like a better option. @LaRiffle @robert-wagner do you know if 2. is possible/worth implementing?